module/metrics/node_info.go

```diff
@@ -0,0 +1,36 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// NodeInfoCollector implements metrics to report static information about node.
+// Such information can include: version, commit, sporkID.
+type NodeInfoCollector struct {
+	nodeInfo *prometheus.GaugeVec
+}
+
+const (
+	sporkIDLabel = "spork_id"
+	versionLabel = "version"
+	commitLabel  = "commit"
```

Comment:
We also have [`ProtocolVersion`](https://github.com/onflow/flow-go/blob/e250dcbb895ce604acc2a535816c647af879d9d2/state/protocol/params.go#L45-L47), which is an integer version number meant to map to the major version for the spork. 

We set this value when bootstrapping sporks now, but don't really use it otherwise. I think this would be a good place to expose it. Then, Metrika could potentially directly compare `version.MajorVersion()` to `ProtocolVersion`, rather than maintaining a separate mapping of `SporkID -> SporkVersionRange`.